README.md

```diff
@@ -67,6 +67,11 @@ In case the error was due to `xcrun: error: unable to lookup item 'PlatformPath'
 
 *Fix:* Switch the default SDK location by running `sudo xcode-select -switch /Applications/Xcode.app/Contents/Developer`.
 
+#### Build configurations
+
+The app is equipped by two custom Build Configurations for ad-hoc distribution over TestFlight and AppCenter.
+The `Development_` prefix added to those two, serves the purpose of picking the correct `Core` module build configuration.
```

Comment:
Quick and clear 👏 

If you think it is worth it, could even link to the ADR.

Reply:
The only reason I woulnd't want to link it is due to this repo being public and people will land into the `404` Github page 😕 . I know it's widely practiced for Jira tickets for other OSS projects but when it comes to Github repo it may be somewhat unpleasant to find into a README doc 🤔